examples/jsm/nodes/pmrem/PMREMUtils.js

```diff
@@ -258,7 +258,7 @@ export const blur = tslFn( ( { n, latitudinal, poleAxis, outputDirection, weight
 
 	const axis = vec3( cond( latitudinal, poleAxis, cross( poleAxis, outputDirection ) ) ).toVar();
 
-	If( all( axis.equals( vec3( 0.0 ) ) ), () => {
+	If( all( axis.equal( vec3( 0.0 ) ) ), () => {
```

Comment:
Hmm... currently `equals` should return a `bvec` while `equal` should return a `bool`.

Reply:
Why `equal` should return `bool`? I think it makes sense to return `bool` in case of primitive types and `bvec` in case of vectors (exactly how `==` works in WGSL and `equal` in GLSL), as now.